pyproject.toml

```diff
@@ -70,6 +70,22 @@ select = ["E4", "E7", "E9", "F", "I"]
 [tool.mypy]
```

Comment:
but we still have the section `[[tool.mypy.overrides]]` ?

Reply:
Yes, this is to harden the rules for the modules we've already enabled typing for.

Reply:
We will still need to work through the modules that still aren't being type checked